cfgen/cfgen

```diff
@@ -1431,51 +1500,54 @@ def pool_drives(m0conf: Dict[Oid, Any],
     if ptype is PoolT.dix:
         cas = [(svc_id, m0conf[svc_id].ctrl_id)
                for svc_id in cluster_svcs(m0conf, SvcT.M0_CST_CAS)]
+        path = '/dev/null'
+        nodes = cluster_desc.get('nodes')
+        if nodes:
+            for nodes in nodes:
+                m0servers = nodes.get('m0_servers')
+                if m0servers:
+                    for m0 in m0servers:
+                        if m0['runs_confd'] is False:
```

Comment:
I think this check is not required, just if the meta data device is present in the cdf then use that.

Reply:
Addressed